Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [lib]
 name = "core"
-crate-type = ["staticlib"]
+crate-type = ["cdylib", "staticlib"]
```

Comment:
The Android JNI requires a dynamic library

Reply:
This will give a warning with the iOS build, but we'll ignore it for the moment.